config/storageClasses.yaml

```diff
@@ -140,6 +140,8 @@ storageClasses:
     pytype: numpy.ndarray
   StrayLightData:
     pytype: lsst.ip.isr.StrayLightData
+  BrighterFatterKernel:
+    pytype: lsst.cp.pipe.makeBrighterFatterKernel.BrighterFatterKernel
```

Comment:
Do you need to make a formatter for this?

Reply:
I have added an entry to config/datastores/formatters.yaml to be safe.  The test I ran ingested a BrighterFatterKernel from the gen2 butler, and that explicitly required a formatter entry (and so didn't cause any issues)